pkg/parse/run.go

```diff
@@ -78,6 +78,7 @@ type RunFunc func(ctx context.Context, r Reconciler, trigger string) RunResult
 
 // DefaultRunFunc is the default implementation for RunOpts.RunFunc.
 func DefaultRunFunc(ctx context.Context, r Reconciler, trigger string) RunResult {
+	klog.Infof("Starting sync attempt (trigger: %s)", trigger)
```

Comment:
After tracing back where `trigger` is set, the trigger would be `reimport` for the periodic reimport event, as well as managementConflicts and watchUpdates events, which makes the log quite confusing. We need to reset the trigger when `opts.HasManagementConflict` or `opts.NeedToUpdateWatch`, or use separate events for managementConflicts and watchUpdates.

Reply:
Yeah, I need to clean the log up here, but I wanted to see if the channels would work first.